### ISSUE TITLE
[Request] Split operation

### ISSUE THREAD

**Issue by ooii** [bnjbvr/kresus#202]
Hi,

It may be useful to split an operation in 2+ categories.
Ex: You pay 100€, 70 as clothes and 30 as food.

Thanks.
PS: I think that merge can also be a good feature but cannot find an example.

**Comment by bnjbvr** [comment]
Thanks for opening the ticket. Do you have a particular use case in mind? Is it for shopping, for instance? Or when you're withdrawing money at the ATM?

**Comment by ooii** [comment]
It may be for payment or withdraw. My personal use-case is when I go to Monoprix and buy clothes and food and pay once. For now, with my own excel sheet, I create to different operations and add a comment to them saying that it's a unique card transaction.

**Comment by bnjbvr** [comment]
I see. So an alternative would be to be able to split an operation into two, but that wouldn't help the automatic duplicate detection that I'm thinking about these days.

**Comment by ooii** [comment]
No, I don't think it would help. In my opinion, that's two distinct problems.

**Comment by ZeHiro** [comment]
Creating suboperations (or adding a "parentOperation" field to the docType could help?
The operations without parentOperation would be managed as they are now, but the one with a parent operation would be ignored
